setup.py

```diff
@@ -78,9 +78,9 @@ def main():
                     'indra.sources.hume', 'indra.sources.hypothesis',
                     'indra.sources.index_cards',
                     'indra.sources.indra_db_rest', 'indra.sources.isi',
-                    'indra.sources.lincs_drug',
-                    'indra.sources.ndex_cx', 'indra.sources.phosphoelm',
-                    'indra.sources.reach',
+                    'indra.sources.lincs_drug', 'indra.sources.ndex_cx',
```

Comment:
In addition to setup.py, we should also add an entry in the README for the new source.